opencti-platform/opencti-front/lang/front/en.json

```diff
@@ -13,10 +13,18 @@
   "Abstract": "Abstract",
   "Accent color": "Accent color",
   "Accept": "Accept",
+  "Access administration": "Access administration",
   "Access already granted": "Access already granted",
+  "Access connectors": "Access connectors",
   "Access CSV feeds directly in your browser": "Access CSV feeds directly in your browser",
+  "Access Dashboards and investigations": "Access dashboards and investigations",
+  "Access dashboards and investigations": "Access dashboards and investigations",
```

Comment:
why do we have these two messages ? Do we have "Access Dashboards and investigations" somewhere ? shouldn't it be only "Access dashboards and investigations" ?

Reply:
problem related to the following